web/src/pages/index.js

```diff
@@ -1,64 +1,74 @@
-import React from 'react'
-import classNames from 'classnames'
+import React from 'react';
+import classNames from 'classnames';
 
-import Nav from '../components/Nav/index'
-import Hero from '../components/Hero'
-import Benefits from '../components/Benefits'
-import Testimonials from '../components/Testimonials'
-import ExampleWaspApps from '../components/ExampleWaspApps'
-import HowItWorks from '../components/HowItWorks'
-import ShowcaseGallery from '../components/ShowcaseGallery'
-import Newsletter from '../components/Newsletter'
-import Faq from '../components/Faq'
-import Footer from '../components/Footer'
-
-import styles from './styles.module.css'
-import './index.css'
-import './preflight.css'
+import Head from '@docusaurus/Head';
```

Comment:
seems like you have some autoformatter that added semicolons to all imports? We don't put them usually, can you pls remove that?